concordium-std/src/traits.rs

```diff
@@ -406,6 +407,17 @@ pub trait HasHost<State>: Sized {
     /// including the amount transferred as part of the invocation.
     fn contract_balance(&self, address: ContractAddress) -> QueryContractBalanceResult;
 
+    /// Get the account's public keys.
+    fn account_public_keys(&self, address: AccountAddress) -> QueryAccountPublicKeysResult;
+
+    /// Verify the signature with account's public keys.
```

Comment:
Since this is the public endpoint, it could use some more documentation of the arguments and outcomes.
Maybe even a small example

Reply:
Added.